ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/SSEConfig.kt

```diff
@@ -15,6 +15,14 @@ import kotlin.time.Duration.Companion.milliseconds
 public class SSEConfig {
     internal var showCommentEvents = false
     internal var showRetryEvents = false
+    internal var allowReconnection = false
+
+    /**
+     * Allows the client to reconnect to the server if the connection is lost.
+     */
+    public fun allowReconnection() {
```

Comment:
Should we consider property instead if method, so it can be overridden in the child config?

Reply:
Maybe this flag can be obsolete with `maxRetries`

Reply:
Rewrite it, so reconnecting will be enabled if `maxReconnectionAttempts` > 0